File: Objects_classes/rectangle.py
#!/usr/bin/python3
class Rectangle:
    def __init__(self, length, width):
        self.len = length
        self.wid = width
    def get_area(self):
        return self.len * self.wid
    def get_perimeter(self):
        return (2 * self.len) + (2 * self.wid)
Rect_1 = Rectangle(10, 20)
print(Rect_1.get_area())
Rect_2 = Rectangle(9, 10)
print(Rect_2.get_area())
print(f"Perimeter is {Rect_1.get_perimeter()}")

File: javascript-classes/rectangle.js
export default class Rectangle{
    constructor(width, height)
    {
        this.width = width;
        this.height = height;
    }
   area()
   {
    return this.width * this.height;
   } 
   perimeter()
   {
    return (this.width + this.height) * 2;
   }
}
const rectangle1 = new Rectangle(10, 20);

const area = rectangle1.area();
const perimeter = rectangle1.perimeter()

console.log(`${area}`);
console.log(`${perimeter}`);